thorlcr/graph/thgraph.cpp

```diff
@@ -2790,20 +2790,33 @@ void CJobBase::init()
 
     logctx.setown(new CThorContextLogger());
 
+    // helpers to preserve legacy behaviour of a few 'expert' properties that could be set as attributes directly under ThorCluster
+    auto getLegacyExpertSettingBool = [this](const char *property, bool dft)
+    {
+        VStringBuffer globalProp("@%s", property);
```

Comment:
I do worry slightly about the overhead of checking multiple levels of options (and the dynamic strings) - but in this case unlikely to have a significant effect.